lib/components/dashboard/screen/dashboardscreen.dart

```diff
@@ -27,80 +29,94 @@ class DashboardState extends State<Dashboard> {
   Widget build(BuildContext context) {
     return BlocBuilder<LoginBloc, LoginState>(
       builder: (context, state) {
-        return Scaffold(
-            floatingActionButton: AddProjectButton(tutorialSettigns),
-            floatingActionButtonLocation:
-                FloatingActionButtonLocation.centerDocked,
-            bottomNavigationBar: Container(
-              decoration: BoxDecoration(
+        if (state is LoggedIn &&
```

Comment:
This one was part of the logging issue I fixed the other day. Ignore this part as well